SingularityUI/app/components/common/formItems/Checkbox.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import FormField from './FormField';
+
+class CheckBox extends React.Component {
+
+  render() {
+    return (
+      <FormField
+        id = {this.props.id}
+        noFormControlClass = {this.props.noFormControlClass}
+        prop = {{
```

Comment:
Gotcha, thanks. I understand that you used `prop` to encapsulate all these settings, but it'd be simpler to have them all as individual props on the component. This something that we can address in a future PR.